applications/StructuralMechanicsApplication/tests/test_dynamic_schemes.py

```diff
@@ -0,0 +1,237 @@
+        node.SetSolutionStepValue(KratosMultiphysics.DISPLACEMENT_Y,0,init_displacement)
+
+        #create element
+        element = mp.CreateNewElement("NodalConcentratedElement3D1N", 1, [1], mp.GetProperties()[1])
+        mass = 1.0
+        stiffness = 10.0
+        element.SetValue(KratosMultiphysics.NODAL_MASS, mass)
+        element.SetValue(StructuralMechanicsApplication.NODAL_STIFFNESS, [0, stiffness,0])
+
+        #time integration parameters
+        time = 0.0
+        end_time = 0.05
+        step = 0
+
+        set_and_fill_buffer(mp,buffer_size,dt)
+
+        #parameters for analytical solution
+        omega = sqrt(stiffness/mass)
+        A = init_displacement
+
+        self.strategy = create_solver(mp, scheme_name)
+
+        # Fill buffer solution
+        for i in range(buffer_size):
+            time = time + dt
+            step = i + 1
+            mp.CloneTimeStep(time)
+            mp.ProcessInfo[KratosMultiphysics.STEP] = step
+            node.SetSolutionStepValue(KratosMultiphysics.DISPLACEMENT_Y, 0, A * cos(omega*time))
+            node.SetSolutionStepValue(KratosMultiphysics.VELOCITY_Y, 0, - A * omega * sin(omega*time))
+            node.SetSolutionStepValue(KratosMultiphysics.ACCELERATION_Y, 0, - A * omega**2 * cos(omega*time))
+
+        # Solve the problem
+        while(time <= end_time):
+            time = time + dt
+            step = step + 1
+            mp.CloneTimeStep(time)
+            mp.ProcessInfo[KratosMultiphysics.STEP] = step
+
+            self.strategy.Solve()
+
+            current_analytical_displacement_y = A * cos(omega*time)
+            current_analytical_velocity_y = - A *omega * sin(omega*time)
+            current_analytical_acceleration_y = - A * omega * omega * cos(omega*time)
+            ## DEBUG
+            #print("DISPLACEMENT_Y:\t", node.GetSolutionStepValue(KratosMultiphysics.DISPLACEMENT_Y,0), "\t",current_analytical_displacement_y)
+            #print("VELOCITY_Y:\t",node.GetSolutionStepValue(KratosMultiphysics.VELOCITY_Y,0), "\t",current_analytical_velocity_y)
+            #print("ACCELERATION_Y:\t",node.GetSolutionStepValue(KratosMultiphysics.ACCELERATION_Y,0), "\t",current_analytical_acceleration_y)
+            # ASSERT
+            self.assertAlmostEqual(node.GetSolutionStepValue(KratosMultiphysics.DISPLACEMENT_Y,0), current_analytical_displacement_y, delta=1e-3)
+            self.assertAlmostEqual(node.GetSolutionStepValue(KratosMultiphysics.VELOCITY_Y,0), current_analytical_velocity_y, delta=1e-3)
+            self.assertAlmostEqual(node.GetSolutionStepValue(KratosMultiphysics.ACCELERATION_Y,0), current_analytical_acceleration_y, delta=1e-3)
+
+    def _base_fall_test_dynamic_schemes(self, scheme_name = "bossak", buffer_size = 2, dt = 1.0e-2):
+        mp = KratosMultiphysics.ModelPart("sdof")
+        add_variables(mp)
+
+        # Create node
+        node = mp.CreateNewNode(1,0.0,0.0,0.0)
+        node.AddDof(KratosMultiphysics.DISPLACEMENT_X)
```

Comment:
it would be good if you add the reactions and check if they are correct (this is just a suggestion, so ignore it if you really don't want)

Reply:
In this problem I don't fix any dof, so no reaction at all